spec/factories.rb

```diff
@@ -112,6 +112,11 @@
     sells "none"
   end
 
+  factory :enterprise_with_images, :parent => :enterprise do
+    logo File.open(File.expand_path('../../app/assets/images/logo-white.png', __FILE__))
+    promo_image File.open(File.expand_path('../../app/assets/images/logo-white.png', __FILE__))
```

Comment:
I would suggest `logo Rails.root.join('app/assets/images/logo-white.png').open` (so it is less sensitive to the location of this file).